rmap-loader-deposit-disco/src/main/java/info/rmapproject/loader/deposit/disco/DiscoDepositConsumer.java

```diff
@@ -139,8 +138,7 @@ public void accept(HarvestRecord record) {
             if (response.getStatusLine().getStatusCode() == 201) {
                 harvestRegistry.register(
                         record.getRecordInfo(),
-                        URI.create(response.getFirstHeader("Location")
```

Comment:
Ah, I see.  At some point, it would be great if it were possible to deploy an RMap for integration test purposes (backed by an in-memory triple store, or something), so that these things can be tested against the actual API.

In any case, what's in this PR looks fine!